skt/runner.py

```diff
@@ -732,11 +698,11 @@ def run(self, url, max_aborted, release, wait=False,
         self.max_aborted = max_aborted
 
         try:
-            job_xml_tree = fromstring(self.__getxml(
-                {'KVER': release,
-                 'KPKG_URL': url,
-                 'ARCH': arch}
-            ))
+            lines = open(self.template).read().splitlines()
```

Comment:
Please also close the file after using it (or use a context manager as the removed function did)

Reply:
```suggestion
            lines = pathlib.Path(self.template).read_text().splitlines()
```

(needs a pathlib import)